src/Cache/Engine/FileEngine.php

```diff
@@ -103,17 +103,6 @@ public function init(array $config = [])
     }
 
     /**
-     * Garbage collection. Permanently remove all expired and deleted data
-     *
-     * @param int|null $expires [optional] An expires timestamp, invalidating all data before.
-     * @return bool True if garbage collection was successful, false on failure
-     */
-    public function gc($expires = null)
```

Comment:
The main purpose of gc() is to garbage collect filesystem caches, which do not have a way to vacuum expired data. We need to retain that behavior or file caching will eventually result in disk exhaustion.